frontend/src/css/NotificationWall.styl

```diff
@@ -72,6 +73,10 @@
   &__highlight
     font-weight bold
 
+// INFO - MB - 20200204 - overrides ListItemWrapper styles because .content is used in many places
+.notification > .notification__list > .content
```

Comment:
Wasn't it possible to rename `.content` to `.notificationWall__content` or something like this?

Reply:
`.content` is also used in the activity feed, it is part of the function `ListItemWrapper`, so it was breaking the design when I tried to change it